templates/server-tls-ca-configmap.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Values.scoop.caConfigMapName }}
+  name: vault-tls-ca
```

Comment:
Use `.Release.Name` in your name to ensure this chart can be installed multiple times.